docker/AccountManager.Dockerfile

```diff
@@ -0,0 +1,15 @@
+# syntax=docker/dockerfile:1
+FROM python:3.9.9-buster
+
+WORKDIR /usr/local/share/
+RUN git clone https://github.com/DarkflameUniverse/AccountManager
+
+WORKDIR AccountManager
+ADD docker/credentials_example.py credentials.py
+ADD docker/resources_example.py resources.py
+RUN pip3 install -r requirements.txt && echo "SECRET_KEY = r'$(openssl rand -base64 30)'" >> credentials.py
```

Comment:
Should we instead have the SECRET_KEY live in the `.env` file so the user can set it?

Reply:
On one hand this would make the portability of the images better. On the other hand, the user would need to create the secret themselves, which is another installation step.

I'll move it to `.env` for the time being.